### PR TITLE
fix for ubuntu 20.04

### PR DESCRIPTION
**Pull request by haxmeadroom**
Fixes Issue with Ubuntu 20.04, which no longer has python-pip

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -12,3 +12,4 @@ pygments
 capstone==4.0.1
 enum34
 pytest
+testresources
```
Comment (zachriggle) [inline requirements.txt:15]: Why is this needed?

Comment (haxmeadroom) [reply requirements.txt:15]: Not sure if it's required, but it got rid of this message:
`ERROR: launchpadlib 1.10.13 requires testresources, which is not installed.  `

**Comment by disconnect3d** [comment]
Tested this branch on `ubuntu:18.04`, `ubuntu:19.10` and `ubuntu:20.04` with this Dockerfile (only changing the image):
```
FROM ubuntu:20.04

ARG DEBIAN_FRONTEND=noninteractive

RUN apt update && apt install -y git gdb && \
	git clone https://github.com/haxmeadroom/pwndbg.git && \
	cd pwndbg && git checkout dev && \
	 ./setup.sh
```

Seems to install & work fine (`docker build -t pwn . && docker run --rm -it --cap-add=SYS_PTRACE pwn gdb /bin/ls`). Thanks!

File: requirements.txt (hunk 1 above)
Comment (disconnect3d) [reply requirements.txt:15]: We might want to move to install this only for ubuntu/debian installs via python3-testresources as in https://github.com/pypa/pip/issues/5372
